backend/src/main/resources/graphql/main.graphqls

```diff
@@ -30,10 +30,6 @@ type Facility {
   phone: String
   email: String
   deviceTypes: [DeviceType!]!
-  defaultDeviceSpecimen: ID
```

Comment:
nit: another mock that would no longer match the response
https://github.com/CDCgov/prime-simplereport/blob/9e3cb43dee5b51bd28b1d2a1c3c60aac28531e12/frontend/src/app/Settings/Facility/FacilityFormContainer.test.tsx#L80-L82

Reply:
thanks for pointing this out, how did you find it? I did a search for all the removed fields and nothing came up, wondering if my system is not searching correctly!

Reply:
Will clean it up as part of a future PR, since it not related to this change set and should have been done when this query was last updated https://github.com/CDCgov/prime-simplereport/blob/9e3cb43dee5b51bd28b1d2a1c3c60aac28531e12/frontend/src/app/Settings/Facility/FacilityFormContainer.tsx#L14-L51